test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -41,6 +41,8 @@ void test_random_iterator(const RandomIt& it) {
         (void) typename RandomIt::iterator_category{};
     }
 
+    RandomIt _it; //Check that RandomIt is default constructible
```

Comment:
Some compilers may get rid of this variable since it's not used. Let's use it anywhere below